### DISCUSSION TITLE
Add custom action button without row selection

### DISCUSSION THREAD

**Discussion by hsiraaj** [rappasoft/laravel-livewire-tables#463] [Feature Requests]
Currently actions or action button comes up with bulk actions. which also enables checkbox for row selection.
Can we have a custom actions without displaying or having to select rows. If it is already possible, can you please share how!

**Comment by rappasoft** [comment]
Just return a view of your actions to the cell:

```
Column::make('Actions')
    ->format(function($value, $column, $row) {
        return view('admin.user.actions')->withUser($row);
    }),
```

**Reply by hsiraaj** [reply]
what if we want to have dropdown action with icons and custom style. not in the cell though..

**Reply by rappasoft** [reply]
Put it above the table?
